API_documentation_Python/Libraries_API_Python.ipynb

```diff
@@ -261,6 +265,155 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
```

Comment:
This section is now duplicated within the notebook - I think a copy/paste went awry.

Reply:
Sorry about that. Completely missed the duplicate section.